### PR TITLE
fix: LiteLLM calls run syncronously by default

### PR DESCRIPTION
**Pull request by anticorrelator**
Due to the complexity of LiteLLM's design, we bypass their async completion as sync completion seems much more stable.

### CODE REVIEW

File: src/phoenix/experimental/evals/models/litellm.py [hunk 1]
```diff
@@ -1,125 +1,127 @@
...
+                top_p=self.top_p,
+                num_retries=self.num_retries,
+                request_timeout=self.request_timeout,
+                **self.model_kwargs,
+            )
+        )
+
+    def _generate_with_retry(self, **kwargs: Any) -> Any:
+        # Bypassing LiteLLM retry logic in preference of our own
+        kwargs.pop("num_retries", None)
```
Comment (anticorrelator) [inline src/phoenix/experimental/evals/models/litellm.py:120]: The change in line feeds has obscured the diff. This is the only line I've added: it removes the automatic LiteLLM retries in preference of our own

**Review by mikeldking — APPROVED** [review]
Strange diff?

**Comment by mikeldking** [comment]
bump

**Comment by anticorrelator** [comment]
Closing for now

**Pull request closed without merging**